src/main/java/controller/LottoController.java

```diff
@@ -16,46 +17,65 @@ public class LottoController {
 
     private final InputView inputView;
     private final OutputView outputView;
-    private int price;
 
     public LottoController(InputView inputView, OutputView outputView) {
         this.inputView = inputView;
         this.outputView = outputView;
     }
 
     public void LottoRun() {
-        price = inputView.inputPrice();
-        LottoMarket lottoMarket = new LottoMarket(price);
-        Lottos lottos = lottoMarket.generateTickets();
-        outputView.printLottoTickets(lottos.getLottos().size());
+        int price = inputView.inputPrice();
+        int passiveLottoCount = inputView.inputPassiveLottoCount();
 
-        generateLotto(lottos);
-        LottoWin(lottos);
+        Lottos autoLottos = generateLottos(price, passiveLottoCount);
+        outputView.printLottoTickets(autoLottos.getLottos().size(), passiveLottoCount);
+
+        List<List<Integer>> passiveLottoNumbers = inputPassiveLottoNumbers(passiveLottoCount);
+        Lottos passiveLottos = generatePassiveLottos(passiveLottoNumbers);
+        Lottos combinedLottos = Lottos.mergeLottos(autoLottos, passiveLottos);
+
+        printLotto(combinedLottos);
+        LottoWin(combinedLottos, price);
     }
 
-    private void generateLotto(Lottos lottos) {
+    private Lottos generateLottos(int price, int passiveLottoCount) {
+        LottoMarket lottoMarket = new LottoMarket(price, passiveLottoCount);
+        return lottoMarket.generateTickets();
+    }
+
+    private List<List<Integer>> inputPassiveLottoNumbers(int passiveLottoCount) {
+        List<String> passiveNumbers = inputView.inputPassiveLottoNumber(passiveLottoCount);
+        List<List<Integer>> lottoNumberLists = new ArrayList<>();
+        for (String passiveNumber : passiveNumbers) {
+            lottoNumberLists.add(new LottoNumberList(passiveNumber).getNumbers());
+        }
+        return lottoNumberLists;
+    }
+
+    private Lottos generatePassiveLottos(List<List<Integer>> lottoNumberLists) {
+        Lottos passiveLottos = new Lottos(new ArrayList<>());
+        return passiveLottos.generatePassiveLottos(lottoNumberLists);
+    }
+
+    private void printLotto(Lottos lottos) {
         for (Lotto lotto : lottos.getLottos()) {
             outputView.printLotto(lotto.getLottoNumbers());
         }
     }
 
-    private void LottoWin(Lottos lottos) {
+    private void LottoWin(Lottos lottos, int price) {
```

Comment:
```suggestion
    private void winLotto(Lottos lottos, int price) {
```

구글 자바 컨벤션에 의하면 메서드는 동사로 시작, camelCase로 네이밍 하도록 나와있습니다!